### DISCUSSION TITLE
Scheduling question

### DISCUSSION THREAD

**Discussion by marceus** [ericaltendorf/plotman#415] [Q&A]
Hi,

I'm trying to plot with the following conditions:
- max 3 plots parallel
- max 1 plot in phase1
- using only 1 temp directory

Please advise if I understand these config parameters:
- tmpdir_stagger_phase_major: 1 _(max 1 plot in phase 1)_
- tmpdir_stagger_phase_minor: 2 _(max 2 plot in phase 2, 3 or 4)_
- tmpdir_stagger_phase_limit: 3 _(max 3 plot per phase)_

When I started plotting using 'plotman plot' it starts the first plot, waits for 1800 sec and starts another one - what have I configured wrong?

**Comment by marceus** [comment]
Ok - after some more reading I now understand what phase_major and phase_minor settings are...
however despite changing settings to the following:
- tmpdir_stagger_phase_major: 2
- tmpdir_stagger_phase_minor: 1
- tmpdir_stagger_phase_limit: 1

Plotman still starts another plot when the first one is in phase 1
Do I need to do anything after changing config? I just run the 'plotman interactive' command...

What am I doing wrong?

**Reply by teamhzipfs** [reply]
Yes. You ctrl+C out of your plotman process and do "plotman plot" again.
